go/vt/vtctl/workflow/server.go

```diff
@@ -127,6 +127,9 @@ const (
 	lockTablesCycles = 2
 	// Time to wait between LOCK TABLES cycles on the sources during SwitchWrites.
 	lockTablesCycleDelay = time.Duration(100 * time.Millisecond)
+
+	SqlFreezeWorkflow   = "update _vt.vreplication set message = '%s' where db_name=%s and workflow=%s"
```

Comment:
We can set the message directly here using the const as we always want to freeze it with the query.

Reply:
done.